var/spack/repos/builtin/packages/r-afex/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RAfex(RPackage):
+    """Analysis of Factorial Experiments
+
+    Convenience functions for analyzing factorial experiments using ANOVA or
+    mixed models. aov_ez(), aov_car(), and aov_4() allow specification of
+    between, within (i.e., repeated-measures), or mixed (i.e., split-plot)
+    ANOVAs for data in long format (i.e., one observation per row),
+    automatically aggregating multiple observations per individual and cell
+    of the design. mixed() fits mixed models using lme4::lmer() and computes
+    p-values for all fixed effects using either Kenward-Roger or Satterthwaite
+    approximation for degrees of freedom (LMM only), parametric bootstrap
+    (LMMs and GLMMs), or likelihood ratio tests (LMMs and GLMMs).
+    afex_plot() provides a high-level interface for interaction or one-way
+    plots using ggplot2, combining raw data and model estimates. afex uses
+    type 3 sums of squares as default (imitating commercial statistical
+    software).
+    """
+
+    homepage = "https://github.com/singmann/afex"
+    cran     = "afex"
+    url      = "https://cran.r-project.org/src/contrib/afex_0.28-1.tar.gz"
```

Comment:
```suggestion
    url      = "https://cloud.r-project.org/src/contrib/afex_0.28-1.tar.gz"
```
Also, note that `url` and `list_url` can be dropped if using the `cran` attribute.